docs/developers/code-reviews.md

```diff
@@ -42,3 +42,4 @@ Some examples include:
 - Does the code use any global, mutable state? [Inject dependencies](https://en.wikipedia.org/wiki/Dependency_injection) instead!
 - Can the code be organized into smaller, more modular pieces?
 - Is there dead code which can be deleted? Or TODO comments which should be resolved?
+- Look for code used by other adapters. Encourage adapter submitter to utilize common code. 
```

Comment:
What is you intent with adding this? This could be interpreted as preferring adapters to import / reference other adapters, but that's not desired.

Reply:
@SyntaxNode - there's been a major push in the Prebid.js world to completely eliminate code duplication. They run an automated scanner that alerts of any copy-paste activity. 

I don't think PBS needs such a hard stance, but obvious copy-paste of whole adapters should be noticed and discouraged. There are several options: either they could utilize aliases, or pull common code out into a library. e.g. on the PBJS side, there are now several adapter libraries. The issue is that there are some 'white labeling' outfits that crank out bid adapters for other companies, often doing 80% of the work with copy-paste. So PBJS has encouraged those companies to up their game with common code libraries, which has cut the amount of adapter code signficantly.